### PR TITLE
Project ready to review

### PR DESCRIPTION
**Pull request by madcido**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        </a>
+      </div>
+      <div class="thumbnail">
+        <a href="">
+          <img src="images/promo_supplier_responsibility.jpg" alt="">
+        </a>
+      </div>
+    </div>
+
+    <div class="footer row align-start">
```
Comment (AdnanAfsari) [inline index.html]: use footer tag instead of Div.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/master.css">
+    <title>Apple-clone</title>
+  </head>
+
+  <body>
+    <img class="bg-img" src="images/your_verse_hero.jpg" alt="">
```
Comment (AdnanAfsari) [inline index.html]: Wrap nav and banner within header tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    <div class="navbar row">
```
Comment (AdnanAfsari) [inline index.html]: use nav tag instead of div.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      <button><img class="logo" src="images/logo.png" alt=""></button>
```
Comment (AdnanAfsari) [inline index.html]: Do you really need to use button tags within a nav? i think its better to use Anchor tag with list items.

File: css/master.css [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+  background: linear-gradient(90deg, rgba(200,200,200,0.2) 0.5%, rgba(0,0,0,0) 2%, rgba(0,0,0,0) 98%, rgba(200,200,200,0.3) 99.5%);
+  color: white;
+  font-size: 13px;
+}
+
+.navbar button:hover{
+  background: radial-gradient(circle, rgba(0, 0, 0, 0.4) 60%, rgba(0, 0, 0, 0.25));
+}
+
+.searchbar{
```
Comment (AdnanAfsari) [inline css/master.css]: remove empty class

**Review by AdnanAfsari — CHANGES_REQUESTED** [review]
Hello @madcido bravo,
You did a wonderful job in this project, Look of the site is perfect, You just have to make few minor changes and i also commented out some.

You have to do some additional work before you submit a project for review and do those in every future projects.

- Update you readme file with project description, Author name and live demo.

- Check your Html file with W3c validator and css file with css lint and make sure your project is error free.

-

**Review by AdnanAfsari — APPROVED** [review]
Hi @madcido 👍 
Great work,  Please go ahead and merge the branch.
Have fun, Happy coding :)

**Comment by madcido** [comment]
@AdnanAfsari thank you, have a nice day aswell =)

**Pull request merged**